.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 7
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: false
+
+# Limit to only `issues` or `pulls`
+only: pulls
```

Comment:
Why not issues? Its safe to say most of issues are stale

Reply:
I’d say our issues aren’t really issues but roadmap items. They aren’t “stale” in the sense that a bug report or pull request becomes stale after a week. StaleBot makes sense if the goal is to ship every open issue or pull request ASAP. 

There isn’t much point in labeling all issues as stale. Makes us look unnecessarily bad. 

Let’s start trying it out on pull requests first!

Reply:
I'd really suggest adding it to issues as well. We can use the exempt labels to override where needed. If we aren't excited to work on it, lets close it. We can create a roadmap in its own issue that is pinned no it won't go stale, in the wiki, or committed with the source. 

I think I agree more with the defaults on the home page for the stalebot repo. Stale should probably be more than a week as that is quite fast for how little has changed on this project recently.

Reply:
Okay, you have convinced me. I have changed the stalebot config to include Issues. I set 2 weeks for issues and 1 week for PRs (since existing work should be looked at actively).

As a first cleanup step, I have marked all issues that I think we could close with the `obsolete?` tag. Here is the full list: https://github.com/Originate/git-town/issues?q=is%3Aissue+is%3Aopen+label%3Aobsolete%3F

Feel free to close the issues where you agree that we are probably not going to get to it, or where the extra complexity isn't worth it.